example/template/demo.html

```diff
@@ -14,13 +14,18 @@
             appToken: 'APP_TOKEN',
             givenName: 'GIVEN_NAME',
             surname: 'SURNAME',
+            email: 'EMAIL',
```

Comment:
not sure how we should handle this page. I keep comment portion of it to test different scenarios.